### PR TITLE
docs: Update info about `graph deploy`

### PR DESCRIPTION
**Pull request by html5cat**
@Jannis done 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,13 +67,13 @@ An example of this can be found in the [Decentraland repository](https://github.
...
-9.  Deploy your subgraph to one of the hosted nodes. The following command builds and deploys the subgraph continuously as you are making changes to it:
+9.  Deploy your subgraph to a Graph Node. The following command builds and deploys the subgraph continuously as you are making changes to it:
     ```sh
     graph \
        --watch \
        --verbosity debug \
-       --node $GRAPH_NODE_MANAGEMENT \
-       --ipfs $GRAPH_NODE_IPFS \
+       --node http://35.242.224.197:8020/ \
+       --ipfs /ip4/35.242.224.197/tcp/8030 \
```
Comment (Jannis) [inline README.md]: One final round please, I should've been more specific. The other example uses `127.0.0.1` so I think that's what we should use her. That way the instructions will work out of the box for people who run everything locally.

Comment (html5cat) [reply README.md]: @Jannis done

**Comment by html5cat** [comment]
Can you merge it? I don’t seem to have permissions

**Comment by Jannis** [comment]
Will do!

**Pull request merged**